app/javascript/stylesheets/components/_notes.scss

```diff
@@ -5,3 +5,8 @@
 .notes-font {
   font-family: 'SF Pro Text';
 }
+
+.delete-alert button:first-child {
+  color: #fff !important;
+  background-color: #2f3941 !important;
+}
```

Comment:
Is this needed? I think we should go ahead with the default Alert styling from neetoUI itself. Did you add this because it was mentioned in the Figma docs?

Reply:
Yedhin I had tried to change this because this was mentioned in the Figma docs. I remember spending considerable time on this, even analysing the source code in the neeto-ui repository to understand how to change the default styling(where the default style was danger). At the end I had to resort to this technique. I request you to kindly update the Figma docs to save the learners of their time.

Reply:
Makes sense. I will assign the update task to someone from the design team.

>to save the learners of their time
>I remember spending considerable time on this, even analysing the source code in the neeto-ui repository to understand how to change the default styling

Your time is not wasted in my opinion. You have learnt something new.

Reply:
Yes Yedhin. I agree with that.